lib/SPIRV/OCLUtil.h

```diff
@@ -543,6 +544,79 @@ getOrCreateSwitchFunc(StringRef MapName, Value *V,
   return addCallInst(M, MapName, Ty, V, nullptr, InsertPoint);
 }
 
+/// Performs conversion from OpenCL memory_scope into SPIR-V Scope.
+///
+/// Supports both constant and non-constant values. To handle the latter case,
+/// function with swtich..case statement will be insterted into module which
+/// \arg InsertBefore belongs to (in order to perform mapping at runtime)
+///
+/// \param [in] MemScope memory_scope value which needs to be translated
```

Comment:
Haven't tested locally, but isn't it a doxygen error if you refer to an argument name that is omitted from the declaration?

Reply:
I will check that, haven't though about this

Reply:
From http://www.doxygen.nl/manual/commands.html#cmdparam:

> The existence of the parameter is checked and a warning is given if the documentation of this (or any other) parameter is missing or not present in the function declaration or definition.

So, it looks like declaration requires parameter names to be specified, will fix it

Reply:
Should be fixed now